mongodb_store/launch/mongodb_store.launch

```diff
@@ -4,6 +4,7 @@
   <!-- This is the outer launch file for mongodb_store. It sets a machine tag for the mongodb_store_inc.launch to use. If you already have a machine tag defined, you can call mongodb_store_inc.launch directly and use the machine argument to it.-->
 
   <arg name="db_path" default="/var/local/mongodb_store"/>
+  <arg name="host" default="" />
```

Comment:
I guess this argument should be passed to the included `mongodb_store.launch`, right?

Also, I suggest renaming it to `mongodb_host`, to be more descriptive.

Reply:
Sorry, I forgot to pass the argument to `mongodb_store_inc.launch`.
I also rename to `mongodb_host`.